enterprise/backend/test/metabase_enterprise/audit_app/pages_test.clj

```diff
@@ -211,9 +211,9 @@
     (mt/with-test-user :crowberto
       (premium-features-test/with-premium-features #{:audit-app}
         (mt/with-temp* [User [a {:email "a@metabase.com" :sso_source nil}]
-                        User [b {:email "b@metabase.com" :sso_source "google"}]
-                        User [c {:email "c@metabase.com" :sso_source "SAML"}]]
-          (is (= ["Email" "Google Sign-In" "SAML"]
+                        User [b {:email "b@metabase.com" :sso_source :google}]
+                        User [c {:email "c@metabase.com" :sso_source :saml}]]
+          (is (= ["Email" "Google Sign-In" "saml"]
```

Comment:
I just noticed that the audit query this is testing (in `metabase-enterprise.audit-app.pages.users`) is still using the `google_auth` field, could you change that as well to only use `sso_source`?

Also I think we should keep that query returning the capitalized names for the SSO methods, since it's user-visible. You might just need to change the case statement to just match on each option directly and return the appropriate name.

Reply:
@qwef you'll need to add cases for JWT and LDAP too so that they're correctly upper-cased; sorry if that wasn't clear